molecule/default/requirements.yml

```diff
@@ -1,6 +1,5 @@
 ---
-- src: kamaln7.swapfile
-  version: "0.4"
+- src: geerlingguy.swap
```

Comment:
@speedlight I think we should target a specific version in order to have more control over the dependencies and gradually upgrade them.

Reply:
@speedlight additionally, I think we need to upgrade the role version

Reply:
Thanks @sestrella, is fixed now.

Reply:
But I need to update the version.. I understand taht with this change is only a minor version update? like 1.6.1, I'm right? @sestrella

Reply:
I push a v.1.6.1 version tag to reflect the change in swap role dependecy.